srtcore/crypto.cpp

```diff
@@ -93,7 +93,7 @@ void CCryptoControl::updateKmState(int cmd, size_t srtlen SRT_ATR_UNUSED)
 
 void CCryptoControl::createFakeSndContext()
 {
-    if (m_iSndKmKeyLen)
+    if (!m_iSndKmKeyLen)
```

Comment:
I think this is already fixed in #365

Reply:
At the time when this fix was submitted, I haven't even started working on #365, so probably yes.